-1

Tested at: e37ad82d67513453b4d3f766714e3a827997086f


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/d19a033fc4765be30cbc94241fc4208d6cd23132
https://github.com/cms-sw/cmssw/commit/06dde7751b4cbc4ba5e08b6d2526be7025e3542c
https://github.com/cms-sw/cmssw/commit/767c57908ef2b6286cf54c962e670f818072ed3f
https://github.com/cms-sw/cmssw/commit/19c3d8de5802b78e93551332faefeb001430e2db
https://github.com/cms-sw/cmssw/commit/beb4bef6b640921976c9d53f16fde9ca7e8519a3
https://github.com/cms-sw/cmssw/commit/f0b49cd572a6b7727c7dc0abd833f8b4e0e6a21e
https://github.com/cms-sw/cmssw/commit/a687825d8e593e871ea885127096a68e6cf4a9e1
https://github.com/cms-sw/cmssw/commit/854267e01a992879d984c8ab9ddbe19f9b8e19b3
https://github.com/cms-sw/cmssw/commit/e5890525ac23f5cc6a3307823bdb05c60efb6b0a
https://github.com/cms-sw/cmssw/commit/cf7c57bdae06f727cd91e17f00dd6765c6f2bbf5
https://github.com/cms-sw/cmssw/commit/17419b88afd2f403cf8a71703f7af097f4909f5d
https://github.com/cms-sw/cmssw/commit/d4b1947b0b3e7160762bee22970672a5d0063d70
https://github.com/cms-sw/cmssw/commit/b11093b6f83e65d2aa192e8d361e94a96ee7cfe1
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22642/26905/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22642/26905/git-merge-result

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22642/26905/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling python modules src/DQM/SiStripMonitorCluster/python
>> Done generating edm plugin poisoned information
>> All python modules compiled
>> Plugins of all types refreshed.
gmake[1]: Leaving directory '/build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_10_1_X_2018-03-14-2300'
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 1
</pre>


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/d19a033fc4765be30cbc94241fc4208d6cd23132
https://github.com/cms-sw/cmssw/commit/06dde7751b4cbc4ba5e08b6d2526be7025e3542c
https://github.com/cms-sw/cmssw/commit/767c57908ef2b6286cf54c962e670f818072ed3f
https://github.com/cms-sw/cmssw/commit/19c3d8de5802b78e93551332faefeb001430e2db
https://github.com/cms-sw/cmssw/commit/beb4bef6b640921976c9d53f16fde9ca7e8519a3
https://github.com/cms-sw/cmssw/commit/f0b49cd572a6b7727c7dc0abd833f8b4e0e6a21e
https://github.com/cms-sw/cmssw/commit/a687825d8e593e871ea885127096a68e6cf4a9e1
https://github.com/cms-sw/cmssw/commit/854267e01a992879d984c8ab9ddbe19f9b8e19b3
https://github.com/cms-sw/cmssw/commit/e5890525ac23f5cc6a3307823bdb05c60efb6b0a
https://github.com/cms-sw/cmssw/commit/cf7c57bdae06f727cd91e17f00dd6765c6f2bbf5
https://github.com/cms-sw/cmssw/commit/17419b88afd2f403cf8a71703f7af097f4909f5d
https://github.com/cms-sw/cmssw/commit/d4b1947b0b3e7160762bee22970672a5d0063d70
https://github.com/cms-sw/cmssw/commit/b11093b6f83e65d2aa192e8d361e94a96ee7cfe1
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22642/26905/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22642/26905/git-merge-result

--- a/Configuration/AlCa/python/autoCond.py
+++ b/Configuration/AlCa/python/autoCond.py
@@ -16,7 +16,7 @@ autoCond = {
     #GlobalTag for MC production with optimistic alignment and calibrations for Run2
     'run2_mc'           :   '101X_mcRun2_asymptotic_v3',
     # GlobalTag for MC production (L1 Trigger Stage1) with starup-like alignment and calibrations for Run2, L1 trigger in Stage1 mode
-    'run2_mc_l1stage1'  :   '93X_mcRun2_asymptotic_v5',
+    'run2_mc_l1stage1'  :   '93X_mcRun2_asymptotic_v4',
     # GlobalTag for MC production (cosmics) with starup-like alignment and calibrations for Run2, Strip tracker in peak mode
     'run2_mc_cosmics'   :   '101X_mcRun2cosmics_startup_deco_v3',
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2

